docs/supported_docker_environment/index.md

```diff
@@ -1,28 +1,25 @@
-# General Docker requirements
+# General Container runtime requirements
 
 ## Overview
 
-Testcontainers requires a Docker-API compatible container runtime. 
-During development, Testcontainers is actively tested against recent versions of Docker on Linux, as well as against Docker Desktop on Mac and Windows. 
+To run Testcontainers-based tests, 
+you need a Docker-API compatible container runtime, 
```

Comment:
```suggestion
To run Testcontainers-based tests, you need a Docker-API compatible container runtime, 
```

Reply:
Oh, the line break here was on purpose, because I use https://sembr.org/ when writing Markdown (generally tends to make diffs easier over time IMO) 😇